docs/source/conf.py

```diff
@@ -85,7 +85,7 @@ def setup(sphinx):
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Defines which version is considered to be the latest stable version.
 # Must be listed in smv_tag_whitelist or smv_branch_whitelist.
-smv_latest_version = 'v1.7'
+smv_latest_version = 'v1.8'
```

Comment:
should you remove 1.8 from `BRANCHES` at this point?

Reply:
good point, removed.